app/services/lom_service/nbp_scrubber.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module LomService
+  class NbpScrubber < Rails::HTML::PermitScrubber
+    ALLOW_LIST = YAML.safe_load_file(Rails.root.join('app/services/lom_service/nbp_scrubber_allow_list.yml'))
+    def initialize
```

Comment:
```suggestion

    def initialize
```

I am still in favor of a new line here, if I may suggest so 🤓